product_margin_classification/models/product_template.py

```diff
@@ -49,15 +49,16 @@ def _compute_theoretical_multi(self):
             classification = template.margin_classification_id
             if classification:
                 multi = 1 + classification.markup
-                if template.taxes_id.filtered(lambda x: x.type != 'percent'):
+                if template.taxes_id.filtered(
+                        lambda x: x.amount_type != 'percent'):
```

Comment:
not a blocking point, but in OCA convention, we avoid to make such changes that are cosmetic. (no reason to write on two lines).

Reply:
Line 52 was longer than 80 characters. So I split it.

Reply:
ok. nice so.